### ISSUE TITLE
Is there a plan to release Kotlin Looker SDK jar?

### ISSUE THREAD

**Issue by Lunaticf** [looker-open-source/sdk-codegen#1042]
I see this...
```
Currently Looker does not publish the Kotlin Looker SDK jar so you need to do
the following steps and copy the resultant jar into the `lib` folder of this project. Build
the Kotlin Looker SDK jar as follows
```

**Comment by jkaster** [comment]
Yes, we do plan to publish the Kotlin SDK but it hasn't been scheduled for a sprint yet. Sorry but we don't have a timeframe for it yet.

**Comment by Lunaticf** [comment]
> Yes, we do plan to publish the Kotlin SDK but it hasn't been scheduled for a sprint yet. Sorry but we don't have a timeframe for it yet.
Sure, Thanks

**Comment by gbotka** [comment]
> Yes, we do plan to publish the Kotlin SDK but it hasn't been scheduled for a sprint yet. Sorry but we don't have a timeframe for it yet.

Do you any update on timeframe?

**Comment by kavanpuranik** [comment]
Any updates on this? It would be so much easier to consume the SDK if it was published in a Maven artefactory just like every other SDK or open source libraries out there, so I am really surprised that your SDK team didn't prioritize it, especially since its a trivial task.

**Comment by daniel-lopes-optimizely** [comment]
@jkaster are there any updates on this matter?

**Comment by daniel-lopes-optimizely** [comment]
@deanlooker is there anything we can do to put this on the near term roadmap?

Our team has to build and place the jar into every project we require, and would like to leverage more of Kotlin scripts to easily provision and manage our Looker Model deployment CI/CD workflows. But Kotlin scripts don't communicate with private maven repositories, and having to pass the jar file into every project will be a tech debt nightmare with every new version coming out. 

If the other language implementations are being published to their respective central package repos, can we also make publishing consistent for Kotlin? It would reduce our tech debt, speed up our upgrade process, and keep dev experience as frictionless as possible.

**Comment by daniel-lopes-optimizely** [comment]
In the meantime, anybody that wants to use a dedicated maven repository, they can set their own deployment github actions project and run gradle with a bespoke init script that injects their Org's maven registry like so:

```bash
cat <<- EOF | >> init.gradle.kts
allprojects {
    version = System.getenv("VERSION_TAG") ?: "0.0.1"
    apply(plugin = "maven-publish")
    afterEvaluate {
        plugins.withId("maven-publish") {
            configure<PublishingExtension> {
                publications {
                    repositories {
                        maven {
                            name = "GitHubPackages"
                            url = uri("https://maven.pkg.github.com/ZaiusInc/looker-kotlin-sdk")
                            credentials {
                                username = System.getenv("GITHUB_ACTOR")
                                password = System.getenv("GITHUB_TOKEN")
                            }
                        }
                    }
                    create<MavenPublication>("javaPublication") {
                        from(components.getByName("java"))
                    }
                }
            }
        }
    }
}
EOF

./gradlew --init-script init.gradle.kts assemble publish
```
